index.html

```diff
@@ -164,10 +164,14 @@ <h2 class="titleekills">Skills</h2>
             <p class="contactparagraph">I'm always interested in hearing about new projects,
                  so if you'd like to chat please get in touch.
             </p>
-            <input class="full-name" type="text" placeholder="Full Name"></br>
-            <input class="email-address" type="email" placeholder="Email address"></br>
-            <input class="enter-text" type="text" placeholder="Enter text here"></br>
-            <button enable="enable" class="get-in-button" type="action">Get in touch</button>
+            <form action="https://formspree.io/f/xeqwoznl" method="post">
+                <ul class="contact-form">
+                    <li><input type="text" id="name" name="full_name" placeholder="   Full Name" maxlength="30" required/></li>
+                    <li><input type="email" id="email" name="contact_email" placeholder="   Email address" required/></li>
+                    <li><textarea id="message" name="contact_message" placeholder="   Enter text here" maxlength="500" required></textarea>
+                    <li class="submit_button"><button type="submit">Get in touch</button></li>
+                </ul>
```

Comment:
Good job with the validations! But let's fix the small design issues, please add padding to left side of input areas like on the template below.
<img width="441" alt="Screenshot 2023-01-24 at 11 17 14 PM" src="https://user-images.githubusercontent.com/25887253/214400853-7337aaff-85e6-48f9-a7c1-a11e9495b902.png">
<img width="441" alt="Screenshot 2023-01-24 at 11 17 59 PM" src="https://user-images.githubusercontent.com/25887253/214400855-5d47a4d7-6fd6-4f3a-b859-6ccd1c7c088f.png">